src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -511,10 +511,6 @@ private function getPropertySchema(PropertyMetadata $propertyMetadata, \ArrayObj
     {
         $propertySchema = new \ArrayObject($propertyMetadata->getAttributes()['swagger_context'] ?? []);
 
-        if (false === $propertyMetadata->isWritable()) {
-            $propertySchema['readOnly'] = true;
```

Comment:
This shouldn't be removed, but [API Gateway](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-known-issues.html) doesn't support it:
>The readOnly field is not supported.